ansible/roles/k8s_install/tasks/k8s_label_nodes.yml

```diff
@@ -0,0 +1,51 @@
+---
+# Copyright 2024-Present, Rackspace Technology, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: "Map node to label/role"
+  ansible.builtin.set_fact:
+    kubectl_options: |
+      {% set _kctl = kubectl_options |default([]) %}
+      {% if (item.hosts |length) > 0 %}
+      {%   for node in item.hosts %}
+      {%     set _label = "" %}
+      {%     set _role = "" %}
+      {%     set _node = "label node " + node + " " %}
+      {%     if item.label is defined %}
+      {%       set _label = item.label %}
+      {%     endif %}
+      {%     if item.role is defined %}
+      {%       set _role = " role=" + item.role %}
+      {%     endif %}
+      {%     if _kctl.append(_node + _label + _role) %}{% endif %}
+      {%   endfor %}
+      {% endif %}
+      {{ _kctl }}
+  with_items:
```

Comment:
Lets change the use of `with_items` to `loop` across the board. While `with_items` works, it's been deprecated.

Reply:
Checking, I looked in the docs and its a bit vague

- We added loop in Ansible 2.5. It is not yet a full replacement for with_<lookup>, but we recommend it for most use cases.
- We have not deprecated the use of with_<lookup> - that syntax will still be valid for the foreseeable future.